addendum.dict

```diff
@@ -369,6 +387,7 @@ Toussaint	𐑑𐑵𐑕𐑨𐑯
 Trafalgar	𐑑𐑮𐑩𐑓𐑨𐑤𐑜𐑼
 Tripadvisor	𐑑𐑮𐑦𐑐𐑩𐑛𐑝𐑲𐑟𐑼
 Trudeau	𐑑𐑮𐑵𐑛𐑴
+Trustpilot 𐑑𐑮𐑩𐑕𐑑𐑐𐑲𐑤𐑩𐑑
```

Comment:
I'd prefer 𐑑𐑮𐑳𐑕𐑑𐑐𐑲𐑤𐑩𐑑 to conform to principle 6 at https://www.shavian.info/spelling/.